docs/01-get-started/installation.mdx

```diff
@@ -1,20 +1,236 @@
 ---
 title: Installation
-description: Installing Front-Commerce
+description:
+  This guide will help you get started with Front-Commerce by installing it.
 sidebar_position: 1
 ---
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+import ContactLink from "@site/src/components/ContactLink";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
 
-Add installation instructions here:
+## Preparation
 
-- ask for an access
-- `npx create-remix my-app --template https://example.com/remix-template.tar.gz`
-- start
-- screenshot!
+This guide will walk you through the steps to configure your system for
+installing Front-Commerce packages from our private npm package repository using
+a global `.npmrc` file.
 
-https://docs.astro.build/en/install/auto/
+### Prerequisites
+
+:::caution
+
+Front-Commerce is a **closed source product**. This documentation supposes that
+you have access to our private repositories.
 
 :::
+
+Access is granted to teams that have signed the relevant legal contracts, either
+as a **Partner** or a **Customer**. Please <ContactLink /> if you need help or
```

Comment:
question: don't we have a time limited trial as well ? shouldn't we mention it there ?

Reply:
Totally. I rephrased to 
> either as a Partner, a Customer or for a Time-Limited Trial.

Thanks!